app/src/androidTest/java/ch/epfl/sweng/SDP/game/drawing/DrawingGameTest.java

```diff
@@ -0,0 +1,42 @@
+package ch.epfl.sweng.SDP.game.drawing;
+
+import android.content.Intent;
+import android.support.test.espresso.matcher.ViewMatchers;
+import android.support.test.rule.ActivityTestRule;
+
+import ch.epfl.sweng.SDP.R;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static org.junit.Assert.*;
+
+public class DrawingGameTest {
+
+    @Rule
+    public final ActivityTestRule<DrawingOffline> activityRule =
+            new ActivityTestRule<DrawingOffline>(DrawingOffline.class){
+
+                @Override
+                protected Intent getActivityIntent() {
+                    Intent intent = new Intent();
+                    return intent;
+                }
+            };
+
+
+    @Test
+    public void getLayoutid() {
```

Comment:
This is not really a test name (testGetLayoutId ?) and also you should use camelback